.pre-commit-config_template.yaml

```diff
@@ -55,72 +57,189 @@ repos:
     docker_image:native: native:ga,from-yml,native:dev,native:maintenance
     docker_image: from-yml
     entry: pylint
+    language: docker
     files: Packs\/.*\.py$
-    exclude: (_test\.py)|(.vulture_whitelist\.py)$
-    args: ['--ignore=demistomock.py,CommonServerPython.py,CommonServerUserPython.py,conftest.py,.venv', '-E', '--disable=bad-option-value,unsubscriptable-object', '-d duplicate-string-formatting-argument', "--msg-template='{path}:{line}:{column}: {msg_id} {obj}: {msg}'", '--generated-members=requests.packages.urllib3,requests.codes.ok']
+    exclude: _test\.py|.vulture_whitelist\.py|test_data
+    args:
+    - --ignore=demistomock.py,CommonServerPython.py,CommonServerUserPython.py,conftest.py,.venv
+    - -E
+    - --disable=bad-option-value,unsubscriptable-object
+    - -d duplicate-string-formatting-argument
+    - "--msg-template='{path}:{line}:{column}: {msg_id} {obj}: {msg}'"
+    - --generated-members=requests.packages.urllib3,requests.codes.ok
     config_file_arg:
-      arg_name: '--rcfile'
-      file_name: '.pylintrc'
-
+      arg_name: --rcfile
+      file_name: .pylintrc
+  - id: pytest-in-docker
+    name: pytest-in-docker
+    description: Run pytest on the code in content packs
+    docker_image:nightly: native:ga,native:maintenance,from-yml
+    docker_image:ci: native:ga,from-yml,native:dev,native:maintenance
+    docker_image:native: native:ga,from-yml,native:dev,native:maintenance
+    docker_image: from-yml
+    entry: python
+    language: docker
+    files: Packs\/.*_test\.py$
+    exclude: test_data
+    args:
+    - /src/Tests/scripts/script_runner.py
+    - pytest
+    - -v
+    - --override-ini='asyncio_mode=auto'
+    - --rootdir=/src
+    - --junitxml=/src/.pre-commit/pytest-junit/.report_pytest.xml
+    - --color=yes
+    - --cov=.
+    - --cov-report=
+    - --files
+    copy_files:
+    - Tests/scripts/dev_envs/pytest/conftest.py
+  - id: pwsh-test-in-docker
+    name: pwsh-test-in-docker
+    description: Run powershell tests
+    entry: pwsh
+    args:
+      - -Command
+      - Invoke-Pester
+      - -Output
+      - Detailed
+      - -CI
+      - -Path
+    files: .Tests.ps1$
+    copy_files:
+      - Packs/Base/Scripts/CommonServerPowerShell/CommonServerPowerShell.ps1
+      - Tests/demistomock/demistomock.ps1
+    split_by_object: true
```

Comment:
run_each_file_seperately: true

lets add this in for now but open a pr either to change the functionality or at least the name